src/session.rs

```diff
@@ -49,13 +50,28 @@ const VERSION: &str = env!("CARGO_PKG_VERSION");
 // appease clippy
 type TsfnOption = Option<Arc<Mutex<ThreadsafeFunction<Vec<String>, ErrorStrategy::Fatal>>>>;
 
+lazy_static! {
+    // Allow user to store a default auth token to use for all sessions
+    static ref AUTH_TOKEN: Mutex<Option<String>> = Mutex::new(None);
```

Comment:
Nice!

Reply:
Heh, copied the 3rd party library behavior there, but it is nice to have.